pages/release_notes.rst

```diff
@@ -5,6 +5,32 @@ Release notes
 
 Whats has been changed, and how to update between versions.
 
+*******
+v0.11.0
+*******
+
+See `github project <https://github.com/orgs/nuts-foundation/projects/5>`_ for more details
+
+=======================
+Features / improvements
+=======================
+
+* A version number has been added to the FHIR consent record which is also visible in the consent-store
+* The consent-store query response has been changed from a simplified model to the complete patient consent model
+* Changed consent on the level of individual resources to data classes across all modules
```

Comment:
Resources and data class are quite generic terms. Can you relate it to the health domain by adding an example?